Letterbook.Docs/_pages/what-is-letterbook.md

```diff
@@ -0,0 +1,7 @@
+---
+title: What is Letterbook
+---
+
+Letterbook is a federated microblogging service, implementing ActivityPub. The goal for the project is to make hosting a fediverse server a better and more sustainable experience. We also want to make it a safer and more inclusive space for the people who make the fediverse their social media home. Those are big aspirations, but we have some solid plans about how to start, and we would love more input about where to go from there.
+
+::: Mastodon https://tech.lgbt/@jenniferplusplus/110567090111682149
```

Comment:
ooh, is this the syntax to embed a mastodon post? That is :fire:

Reply:
Yeah, I added a mastodon embed feature, seemed handy.  Mastodon uses an embed script (ex: https://tech.lgbt/embed.js, https://hachyderm.io/embed.js) and I think I'll pull that into the project itself to simplify things a bit - but it's kind of messy. The embed script is slightly different across servers (though I don't think significantly so). It also probably only works with Mastodon clients right now - Sharkey won't work for example.